packages/store/src/utils/app-studio.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @todo: This needs to be replaced by the functions from `@sap-ux/btp-utils`
+ */
+
+/**
+ * Enumeration of environment variables used in AppStudio
+ */
+ export enum ENV {
+    PROXY_URL = 'HTTP_PROXY',
+    H2O_URL = 'H2O_URL'
+}
+
+export function isAppStudio(): boolean {
```

Comment:
Is store the right place for this function?

Reply:
Good catch. I should use btp-utils here.

Reply:
`btp-utils` is part of https://github.com/SAP/open-ux-tools/pull/211
I'll replace it when that PR is merged in. Going to leave it in for now.